### DISCUSSION TITLE
Plan for Release

### DISCUSSION THREAD

**Discussion by jonmiller3** [jonmiller3/Generalization_Bound_Toolbox#15] [General]
There should be a Release Branch/Fork and an Experimental Branch/Fork. Initialization the Optimization Bound should be in the Experimental and not the Release. This should be setup by October 24.

We need to fix the bug in gen_ztw_c, it should produce the same result as gen_ztw.

We should include another example and more words explaining use for the Jupyter notebooks. These can serve as documentation and examples.

We should make clear in the Readme the the domain of the target functions.

We should improve the Unit Tests.

We should come up with a list of things to be done. Jon will then put a timeline to do them. The desired release date is by the end of September.

**Comment by bnord** [comment]
Why Oct 24?

**Reply by jonmiller3** [reply]
It is after my deadline of October 23.

**Comment by bnord** [comment]
Will there be issues for each item?

**Reply by jonmiller3** [reply]
Items that require changes to the code or additions will be given issues. Organizational items, I think, should not be given issues.
